test/e2e/arktos/multi_tenancy/test_suites/crd/test_system_share_crd.yaml

```diff
@@ -0,0 +1,160 @@
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ System Forced Sharing CRD Tests ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+# This test suite verifies the system forced sharing CRD works in Arktos
+#~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+######################################################
+# test variables
+######################################################
+Variables:
+  first_tenant: random_8
+  second_tenant: random_8
```

Comment:
why are first and second tenants the same?

Reply:
random_8 means a randomly generated string of length 8. So they will be different.

Reply:
got it.